packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx

```diff
@@ -114,10 +112,58 @@ export const MeetingCallControlBar = (props: MeetingCallControlBarProps): JSX.El
               data-ui-id="meeting-composite-people-button"
               disabled={props.disableButtonsForLobbyPage}
               label={meetingStrings.peopleButtonLabel}
+              styles={commonButtonStyles}
+            />
+          )}
+          {isEnabled(meetingCallControlOptions?.chatButton) !== false && (
+            <ChatButton
+              checked={props.chatButtonChecked}
+              showLabel={true}
+              onClick={props.onChatButtonClicked}
+              data-ui-id="meeting-composite-chat-button"
+              disabled={props.disableButtonsForLobbyPage}
+              label={meetingStrings.chatButtonLabel}
+              styles={commonButtonStyles}
             />
           )}
-        </Stack.Item>
+        </Stack>
       )}
     </Stack>
   );
 };
+
+const buttonContainerStyle: IStyle = {
+  padding: '0.75rem',
+  columnGap: '0.5rem'
+};
+
+const controlBarStyles: BaseCustomStyles = {
+  root: buttonContainerStyle
+};
+
+const commonButtonStyles: ControlBarButtonStyles = {
+  root: {
+    border: 'solid 1px #E1DFDD',
+    borderRadius: '0.25rem',
```

Comment:
You can use `theme.effects.roundedCorner4` here

Reply:
Decided not to use theme here because the `ControlBarButton` explicitly styles with `borderRadius: 0` and `EndCallButton` explicitly styles with `borderRadius: 0.5rem`.

I'd like these to stay consistent (applying theme halfway would be worse, imo)

Reply:
But let me take another holistic look at using theme. I'll use theme if the base buttons are using them.

Reply:
Done.